.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
OK that's interesting... I wonder why that happens.

Thanks for the extra detail... as I mentioned before, I think it would be great to add this extra context into a comment in `.travis.yml` file for the benefit of future contributors (and our future selves!)